[2024-03-12T11:14:15.302+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.insert_into_table manual__2024-03-12T11:09:11.280916+00:00 [queued]>
[2024-03-12T11:14:15.324+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.insert_into_table manual__2024-03-12T11:09:11.280916+00:00 [queued]>
[2024-03-12T11:14:15.325+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 6
[2024-03-12T11:14:15.366+0000] {taskinstance.py:2214} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-03-12 11:09:11.280916+00:00
[2024-03-12T11:14:15.381+0000] {standard_task_runner.py:60} INFO - Started process 2454 to run task
[2024-03-12T11:14:15.401+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v02', 'insert_into_table', 'manual__2024-03-12T11:09:11.280916+00:00', '--job-id', '890', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpaphhuu8r']
[2024-03-12T11:14:15.409+0000] {standard_task_runner.py:88} INFO - Job 890: Subtask insert_into_table
[2024-03-12T11:14:15.584+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v02.insert_into_table manual__2024-03-12T11:09:11.280916+00:00 [running]> on host 26e63b1d0d39
[2024-03-12T11:14:15.826+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='daniel' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v02' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-03-12T11:09:11.280916+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-12T11:09:11.280916+00:00'
[2024-03-12T11:14:15.828+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-03-12', 'dag_with_postgres_operator_v02')
        
[2024-03-12T11:14:15.859+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-12T11:14:15.892+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-12T11:14:15.911+0000] {sql.py:457} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-03-12', 'dag_with_postgres_operator_v02')
        , parameters: None
[2024-03-12T11:14:15.917+0000] {sql.py:466} INFO - Rows affected: 1
[2024-03-12T11:14:15.931+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-03-12T11:14:15.956+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v02, task_id=insert_into_table, execution_date=20240312T110911, start_date=20240312T111415, end_date=20240312T111415
[2024-03-12T11:14:16.022+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-12T11:14:16.142+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
